androidlibrary_lib/src/main/java/org/opendatakit/builder/InitializationUtil.java

```diff
@@ -54,6 +54,9 @@
 
   private static final String TAG = "InitializationUtil";
 
+  private static final String csv = "csv";
```

Comment:
Java convention is to capitalize constant variable names. Please capitalize the variables so it is CSV and PROPERTIES